manim/mobject/mobject.py

```diff
@@ -1021,7 +1021,13 @@ def rotate_about_origin(self, angle, axis=OUT, axes=[]):
         """Rotates the :class:`~.Mobject` about the ORIGIN, which is at [0,0,0]."""
         return self.rotate(angle, axis, about_point=ORIGIN)
 
-    def rotate(self, angle, axis=OUT, **kwargs):
+    def rotate(
+        self,
+        angle,
+        axis=OUT,
+        about_point: Union[np.ndarray, List, None] = None
```

Comment:
missing a comma here
```suggestion
        about_point: Union[np.ndarray, List, None] = None,
```